examples/stable_diffusion/README.md

```diff
@@ -0,0 +1,33 @@
+## Setup
+
+1. Install skypilot by following these [instructions](https://skypilot.readthedocs.io/en/latest/getting-started/installation.html)
+
+2. Run sky launch -c stable-diffusion stable_diffusion_docker.yaml
```

Comment:
perhaps add a block git clone and cd to this directory:
```
git clone https://github.com/skypilot-org/skypilot.git && cd examples/stable_diffusion
sky launch -c stable-diffusion stable_diffusion_docker.yaml
```

The idea is to have a copy-paste-able readme :)